### ISSUE TITLE
doodles reportedly don't work in mobile brave

### ISSUE THREAD

**Issue by MightyPork** [glitch-soc/mastodon#225]

**Comment by nightpool** [comment]
might be the same as FF ESR modal bug?

**Comment by noiob** [comment]
If this is Brave on iPhone it might be the same as the fact that it doesn't work in Safari (both mobile and desktop), since all iOS browsers are pretty much forced to use the built-in webkit

**Comment by chriswmartin** [comment]
Just tested doodles in Brave on Android and they work, this seems specific to iPhone

**Comment by marrus-sh** [comment]
Yeah I'm gonna guess that this is another manifestation of the same bug as #169
